testdata/p4_16_samples/psa-action-profile1.p4

```diff
@@ -0,0 +1,100 @@
+#include <core.p4>
```

Comment:
Maybe we should name these files with -psa.p4 as a suffix.

Reply:
I am using psa-*-bmv2.p4 for psa program that are ready for bmv2 test.
and psa-*.p4 for psa program that are only passing frontend/midend.